src/Chapter1.hs

```diff
@@ -556,8 +560,10 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
-
+isVowel :: Char -> Bool
+isVowel c
+  | c == 'a' || c =='e' || c =='i' || c == 'o' || c == 'u' = True
+  | c /= 'a' || c /='e' || c /='i' || c /= 'o' || c /= 'u' = False
```

Comment:
The negation of the previous case is not `c /= 'a' || c /='e' || ...` but `c /= 'a' && c /='e' && ...`.

But actually you don't need that all together, as `otherwise` is exactly covers what doesn't suits for the first case 🙂 
```suggestion
  | otherwise = False
```